lttoolbox/att_compiler.cc

```diff
@@ -52,21 +54,22 @@ AttCompiler::clear()
  *       for conversion?
  */
 void
-AttCompiler::convert_hfst(wstring& symbol)
+AttCompiler::convert_hfst(UString& symbol)
 {
-  if (symbol == L"@0@" || symbol == L"ε")
+  if (symbol == "@0@"_u || symbol == "ε"_u)
```

Comment:
`"ε"_u` probably better written as `u"\u03B5"_u`

Reply:
yep, and I plan to move these to a list of constants anyway